functions/src/events/helpers.test.ts

```diff
@@ -0,0 +1,21 @@
+import { subDays } from "date-fns"
+import { withinCutoff } from "./helpers"
+
+describe("withinCutoff", () => {
+  beforeEach(() => {
+    jest.useFakeTimers()
+  })
+
+  afterEach(() => {
+    jest.useRealTimers()
+  })
+
+  it("should return true for a date within three days", () => {
```

Comment:
I think we should also make sure to cover the cases we suspect were missed in the last pass:
* startsAt before the cutoff (e.g. 10 days ago)
* startsAt after now (e.g. 2 days in the future)

Reply:
Added a future negative check in the next commit